src/inference_profiling/local_model_tests_visualization.ipynb

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
<p>What is peak_proc_cpu and peak_proc_mem useful for? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
The "peak_proc_cpu" and "peak_proc_mem" gives the peak process cpu and proess memory utilization of a particular program that is being profiled.